module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php

```diff
@@ -91,325 +101,103 @@ public function __invoke(
         if (!empty($options)) {
             throw new \Exception('Unexpected options sent to factory.');
         }
+        $specPluginManager = $container->get(\VuFind\RecordDataFormatter\Specs\PluginManager::class);
+        // for backward compatability check if getDefault*Specs methods got overridden.
```

Comment:
Minor spelling correction:

```suggestion
        // for backward compatibility check if getDefault*Specs methods got overridden.
```